### ISSUE TITLE
Add monocrome tray icons

### ISSUE THREAD

**Issue by geser2008** [keshavbhatt/whatsie#93]
Hello.
Please add monochrome tray icons for light and dark panel background.
I'm running v.4.9 build from AUR on Manjaro.
Thank you.

**Comment by Sadi58** [comment]
My amateur DIY attempts: 😉

1. I first tried the easy way: replacing some color app icons with monochrome ones, but it didn't work. The _hicolor_ icon theme (_/usr/share/icons/hicolor_) has a color icon named _com.ktechpit.whatsie_ in five different sizes as well as _scalable_, but also a monochrome icon named _com.ktechpit.whatsie-symbolic_, but they are not used in systray (so, replacing those color icons or adding monochrome icons to _~/.local/share/icons/hicolor_ has no effect).

2. I then tried building Whatsie from source after replacing _whatsapp-message-32.png_ and _whatsapp-message.svg_ in subdirectory _./src/icons/app/_ with monochrome ones, and doing the same for all icons in subdirectory _./src/icons/app/notification/_ which made some difference as can be seen in the screenshot below! The only problem now is that number of new messages got stuck at 1, but maybe it's a one-off thing, and maybe it will work next time I have several several unread messages. I'm also attaching the monochrome icons I've used in a compressed file below.
![whatsie-notify](https://github.com/keshavbhatt/whatsie/assets/6375934/1ff24bd0-0b61-43dc-a1c9-bc04622b72c0)
[whatsie-src-icons-app.tar.gz](https://github.com/keshavbhatt/whatsie/files/13318388/whatsie-src-icons-app.tar.gz)

**Comment by tubbadu** [comment]
In case you still need this, changing all icons `icon-XY.png` and also all icons at `whatsie/src/icons/app/notification/whatsie-notify-XY.png` managed to get a monochrome icon:
![image](https://github.com/keshavbhatt/whatsie/assets/56864069/0c02668f-cd20-44d5-bc75-83e56d132a11)

here's my icons, you can change the notification bubble color and the icon color to make it suitable for your theme if you want to: 
[app.tar.gz](https://github.com/keshavbhatt/whatsie/files/13370276/app.tar.gz)
just replace the whole `whatsie/src/icons/app` dir with the content of the tar.gz and build it

**Comment by Sadi58** [comment]
@tubbadu I think we only need to change the icons in subdirectory `./src/icons/app/notification/` to get monochrome icons in systray, and those in `./src/icons/app/` are app icons, which might be left intact - even including `whatsapp-message-32.png` and `whatsapp-message.svg` which probably change the app icon (when Whatsie window is not minimized, i.e. displayed in Task Manager). Am I right?

**Comment by allan-null** [comment]
It would be really nice if this app would get its icons from the icon theme.

**Comment by Sadi58** [comment]
@allan-null Yes, it would be much better if the status icons were placed in e.g. "/usr/share/icons/hicolor/22x22/status" just like the app icons placed in "/usr/share/icons/hicolor/[64x64|128x128|512x512]/apps". This would also make it much easier to use monochrome dark/light variations as well.

**Comment by shakaran** [comment]
Good news: this is implemented in the maintained fork **Whatly**. There's a monochrome tray-icon option in Settings, and the icon also carries the unread count and dims when disconnected. https://github.com/shakaran/whatly

cc @geser2008 @Sadi58 @allan-null @tubbadu

**Comment by keshavbhatt** [comment]
The latest version (6.0.1) has this: open Settings and, under "System tray", enable "Use a monochrome (symbolic) icon".
To update to the latest version:
- Snap: `sudo snap refresh whatsie`
- Flatpak: `flatpak update com.ktechpit.whatsie`

Closing as done.
